### PR TITLE
Add bitcoin-core-24.0 to bitcoin-magazine/bitcoin-explained

### PR DESCRIPTION
**Pull request by realdezzy**
This PR adds [Bitcoin Core 24.0](https://www.youtube.com/watch?v=3UfrB7_ZOx0) transcript to the bitcoin-magazine/bitcoin-explained directory.

### CODE REVIEW

**Comment by realdezzy** [comment]
@kouloumos please take a look at the changes I've made here also

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 1]
```diff
@@ -0,0 +1,1150 @@
...
+
+I'm good.
+
+Sjors Provoost: 00:00:27
+
+How do you like the weather?
+
+Aaron van Wirdum: 00:00:29
+
+Yeah, well, It was too hot all summer and then it was nice for about a week and now we're back to too cold.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:39]: Removing "So..." form the end here is correct.

You could also remove the leading filler words. In general you should remove introductory filler words ('Like', 'So', 'Right') that start sentences but don’t contribute to the overall meaning
```suggestion
It was too hot all summer and then it was nice for about a week and now we're back to too cold.
```

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 2]
```diff
@@ -0,0 +1,1150 @@
...
+
+So, there are a bunch of new features as well as bug fixes and improvements, sorry, performance improvements in this new release.
+We're going to discuss a couple of these.
+This is kind of just our selection.
+If this podcast was made by two other people, by Ors and Sharon, they might make a different selection.
+But we, Sjors and Aaron, we made the selection.
+
+Sjors Provoost: 00:04:59
+
+Yeah, I mean, you can look at all the commits, so all the atomic changes that go into a release, and there could be a thousand of them, but most of them are very boring, or at least they are very boring unless you are really into details of how compilers work or into cleaning up code for the sake of cleaning it up.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:153]: Another example of a filler woord/phrase at the beginning of the sentece. 
"Yeah, I mean" don't add significant meaning to the sentence and can be removed.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 3]
```diff
@@ -0,0 +1,1150 @@
...
+So there's a lot of changes that are just not very interesting to discuss, although they are important and they should be, they are being, you know, they always get reviewed because any of those changes could of course be a really scary thing.
+
+Aaron van Wirdum: 00:05:29
+
+Yeah, I'm just saying we made a selection of some of the changes that we think are probably the most interesting
+to you, our dear listener.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:159]:
```suggestion
 I'm just saying we made a selection of some of the changes that we think are probably the most interesting to you, our dear listener.
```

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 4]
```diff
@@ -0,0 +1,1150 @@
...
+
+Sjors Provoost: 00:06:41
+
+Plus a little bit of extra info that, for example, contains the timestamp and the amount of proof of work.
+Well, the amount of proof of work follows in the hash.
+
+Aaron van Wirdum: 00:06:52
+
+It's basically hash plus some extra info.
+Yes.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:208]: This "Yes" was misattributed to Aaron. There are instances where phrases like 'Right' or 'Yeah' from one speaker mistakenly appear in the text attributed to another speaker. Please remove these occurrences for clarity and accuracy.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 5]
```diff
@@ -0,0 +1,1150 @@
...
+So it's not the whole block.
+Okay.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:210]: This "Okay" here is an example of a non-essential line that can be deleted.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 6]
```diff
@@ -0,0 +1,1150 @@
...
+
+Okay, that was already the case.
+So even in Bitcoin Core 23, you would already start with downloading only the headers and only later the blocks.
+Okay, so what has changed?
+
+## Download headers twice
+
+Sjors Provoost: 00:07:41
+
+So what has changed now is that we're going to download the headers twice.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:230]: The word 'So' is frequently used as a filler at the beginning of sentences, often without adding meaningful content. While it occasionally serves a purpose, in many cases, removing it enhances readability.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 7]
```diff
@@ -0,0 +1,1150 @@
...
+
+Right, yeah, a checksum is essentially you add up all kinds of numbers and you get a very short number which doesn't in itself prove anything, but I guess if you do it a bunch of times then.
+
+Sjors Provoost: 00:13:46
+
+If you add up the same numbers you're gonna get the same checksum.
+
+Aaron van Wirdum: 00:13:48
+
+Right, exactly.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:428]: This kind of lines don't contribute to the transcript's content - you can just delete this and continue with the same speaker. This transcript has a lot back n forth between participants, so you'll notice the presence of numerous filler words or non-essential lines that need removal.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 8]
```diff
@@ -0,0 +1,1150 @@
...
+So they may get, in the long run, they may get a better lightning experience out of this.
+
+Aaron van Wirdum: 00:21:36
+
+I mean, my personal opinion is that this should just be the general rule.
+Transactions should always be broadcast and forwarded, especially if they have a higher fee.
+
+Sjors Provoost: 00:21:48
+
+I mean, maybe, we've already had this discussion during that episode, I guess, but I would also say that if you just turn on the RBF flag by default, it shouldn't bother you either.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:592]: You'll notice that Sjors frequently uses 'I mean' as a filler phrase to start his sentences.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 9]
```diff
@@ -0,0 +1,1150 @@
...
+
+Sjors Provoost: 00:22:52
+
+Yeah, I'm not sure how much we covered the descriptor wallets so far but the gist of it is that the Bitcoin Core wallet is quite old.
+
+It used to be just a bag of keys and then given a private key, the wallet would have to pay attention to certain scripts.
+I think in one of the first episodes we explained what scripts are.
+
+But the simplest scripts is just that anybody with the public key can spend this coin.
+The second simplest script is that anybody with the hash of the public key, like, you know, can spend the coin or anybody with the, you know, the public key for which the hash is blah, blah, blah.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:618]: Here ",like, you know," is an example of a filler word/phrase - eliminating it helps with readability
```suggestion
The second simplest script is that anybody with the hash of the public key, can spend the coin or anybody with the, the public key for which the hash is blah, blah, blah.
```

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 10]
```diff
@@ -0,0 +1,1150 @@
...
+Because you had the address of BC1, and you had the address that looked like a P2SH address with SegWit wrapped in it.
+So the wallet just became a giant mess.
+
+Aaron van Wirdum: 00:23:42
+
+Because you have multi-sig writes and stuff like that.
+
+Sjors Provoost: 00:23:45
+
+Well, yeah, that makes it even more complicated.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:630]: The word "Well," is also often used as a filler word to start a sentence.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 11]
```diff
@@ -0,0 +1,1150 @@
...
+
+Sjors Provoost: 00:33:41
+
+Or if there's one input.
+
+Sjors Provoost: 00:33:44
+
+Let's say you, well, if there is, depends on how much there is available, right?
+But you're looking at the chain, so you don't know what was available.
+But if your wallet is looking at a bag of coins, it might, you know, start with the smallest coin saying, nah, it's not big enough.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:956]: Another occurrence of ", you know," as a filler.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 12]
```diff
@@ -0,0 +1,1150 @@
...
+Sjors Provoost: 00:28:00
+
+No, well, yeah, in order to create the miniscript itself, you will need separate software.
+There is a miniscript compiler, but you can write miniscript by hand too if you want to.
+So, in principle, you don't need a second piece of software.
+Now, the idea is that a future version of Bitcoin Core will also be able to sign for it.
+
+Aaron van Wirdum: 00:28:16
+
+Sure, yeah.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:774]: Another example of a non-essential line.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 13]
```diff
@@ -0,0 +1,1150 @@
...
+Now you don't actually need to include the flag anymore.
+If this setting is turned on?
+
+Sjors Provoost: 00:18:44
+
+Yeah, I think that if you turn this setting on, then the flag does no longer have to be in there, it'll relate anyway so then I'm guessing there are other rules though because I assume the fee has to go up but I'm not sure about that
+
+Aaron van Wirdum: 00:18:56
+
+So in my case if so far I was running a Bitcoin Core node let's say what was the previous one 23 of course So far I was running Bitcoin Core 23 and then my node would not forward a transaction even if it had a higher fee and it conflicted with the previous transaction.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:543]: Here you can see a self-corrections where the speaker revises their own words for accuracy or to add detail. In this case you should include only the most clear and final version of what they say.
```suggestion
So in my case, if so far I was running Bitcoin Core 23, my node would not forward a transaction even if it had a higher fee and it conflicted with the previous transaction.
```

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md [hunk 14]
```diff
@@ -0,0 +1,1150 @@
...
+It would not do that.
+But now with Bitcoin 24 I can switch the setting and now it will actually forward that transaction.
+
+Sjors Provoost: 00:19:22
+
+Yeah, I think that's it and it basically means that if you're relying on this opt-in RBF system to prevent double spends, basically to prevent double spends, you should now rely on that less so, because there's going to be more nodes that will relay this thing regardless.
+
+Aaron van Wirdum: 00:19:37
+
+Yeah, well, I mean, it was almost, well, I don't know about almost certainly, but there's definitely a much bigger chance that a conflict in a transaction will make it to a miner now, right?
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:553]: Here is an example of sentence where there is a lot of filler in the beginning.
```suggestion
I don't know about almost certainly, but there's definitely a much bigger chance that a conflict in a transaction will make it to a miner now, right?
```

**Review by kouloumos — COMMENTED** [review]
Great job on the edits! 👍 I really appreciate how you've handled the technical terms with backticks and trimmed out some of those "Yeah" and "Right" fillers – it’s these little things that really polish up a transcript.

I've left a few comments inline with some examples where we could polish things even further. Please have a look at my comments and the feedback I provided on [your other review](https://github.com/bitcointranscripts/bitcointranscripts/pull/362#issuecomment-1933717670). A bit more fine-tuning based on the feedback, and we should be good to go.

Thanks so much for your patience and for the effort you've put in. I know I've been a bit slow on the uptake here, and I really appreciate your understanding.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md (hunk 13 above)
Comment (kouloumos) [reply bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:543]: This is one of the fixes that really improves readability

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md (hunk 14 above)
Comment (kouloumos) [reply bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md:553]: this one also, and also an easy catch during review.

**Comment by kouloumos** [comment]
Fantastic work, @realdezzy! Your editing skills continue to impress. As mentioned in your other review, we're tweaking our review approach, but I won't delve into that here since you're already up to speed.

I've just added [some minor fixups](https://github.com/bitcointranscripts/bitcointranscripts/pull/364/commits/812ef60261759220f24290d3a82559b43f40a9af) and now your submission is all set to merge – great job! When you get a chance, please take a look at those changes. Notice how I adjusted a few headers to serve as subheaders under related topics. This approach can enhance the structure and flow of the content.

Your attention to detail and commitment to quality are truly making a difference. I'm excited to see your next review. Keep up the amazing work!

**Pull request merged**